pkg/config/defaults.go

```diff
@@ -159,6 +159,19 @@ const (
 	LakefsEmailBaseURLKey      = "email.lakefs_base_url"
 
 	UIEnabledKey = "ui.enabled"
+
+	GravelerRepositoryCacheSizeKey       = "graveler.repository_cache.size"
+	DefaultGravelerRepositoryCacheSize   = 1000
+	GravelerRepositoryCacheExpiryKey     = "graveler.repository_cache.expiry"
+	DefaultGravelerRepositoryCacheExpiry = 5 * time.Second
+	GravelerRepositoryCacheJitterKey     = "graveler.repository_cache.jitter"
+	DefaultGravelerRepositoryCacheJitter = DefaultGravelerRepositoryCacheExpiry / 2
+	GravelerCommitCacheSizeKey           = "graveler.commit_cache.size"
+	DefaultGravelerCommitCacheSize       = 50000
+	GravelerCommitCacheExpiryKey         = "graveler.commit_cache.expiry"
+	DefaultGravelerCommitCacheExpiry     = 30 * time.Second
```

Comment:
Nice... but even this seems low!  What's the harm in keeping something in the commit cache?  Even 5 minutes seems reasonable.

Reply:
No harm - updated the size and forgot to update the expiry. Setting to 10min - comment if you think we should go higher.